src/System.DirectoryServices/ref/System.DirectoryServices.csproj

```diff
@@ -3,11 +3,14 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{CBCDA53B-4C01-4267-B08C-413205FE4D8D}</ProjectGuid>
+    <!-- UAPvNext is not yet mapped to netstandard2.0, manually duplicate this ref -->
```

Comment:
@joperezr correct me if I'm wrong but I wouldn't expect this to be needed any longer. Nuget should have the mapping now.

Reply:
Just talked to @joperezr offline and he said that this is still needed as for the versions that we are still using the mapping is not there yet.

Reply:
I doubt we are using the nuget dlls that have the mappings in place yet, so I do believe this is required still.